bounties/packagist/monica/3/vulnerability.json

```diff
@@ -0,0 +1,58 @@
+{
+    "PackageVulnerabilityID": "3",
+    "DisclosureDate": "2020-09-02",
+    "AffectedVersionRange": "*",
+    "Summary": "Xss at Multiple inputs triggered by a single endpoint",
+    "Contributor": {
+        "Discloser": "B3EF",
+        "Fixer": ""
+    },
+    "Package": {
+        "Registry": "packagist",
+        "Name": "monica",
+        "URL": "https://app.monicahq.com/settings/auditlogs",
+        "Downloads": ""
+    },
+    "CWEs": [
+        {
+            "ID": "CWE-725",
+            "Description": "Cross-Site scripting"
+        }
+    ],
+    "CVSS": {
+        "Version": "3.1",
+        "AV": "",
+        "AC": "",
+        "PR": "",
+        "UI": "",
+        "S": "",
+        "C": "",
+        "I": "",
+        "A": "",
+        "E": "",
+        "RL": "",
+        "RC": "",
+        "Score": "7.x"

```

Comment:
The CVSS needs to be updated with the exact value (used to identify the number of credits awarded to you and the fixer).
It's more a `self-stored XSS` in my opinion, and that's why I would suggest lowering it a bit, like `5.6`

Reply:
I anyway make it `medium` since it's triggered by multiple `endpoints` and chaining it with a `CSRF login` could make possible `persistent open redirect/deface` :smile:

Cheers,
Mik

Reply:
I agree with @Mik317 on this one - I'd give it a 5.2